src/main/java/com/amadeus/resources/FlightOfferSearch.java

```diff
@@ -16,8 +16,10 @@ protected FlightOfferSearch() {}
   private @Getter String id;
   private @Getter String source;
   private @Getter boolean instantTicketingRequired;
+  private @Getter boolean disablePricing;
```

Comment:
![Screenshot 2022-03-23 at 17 41 54](https://user-images.githubusercontent.com/15012967/159751513-f1c95c0a-1f8a-4ba6-aefa-dc0ea2ca8ada.png)
Why don't you create the new resource for the API?

Reply:
Hello Anna! From my understanding, the picture you show above is the input body for this API.

Reply:
@tsolakoua Hello! From my understanding, I think the picture you commented on above is the input body for this API.

Reply:
After testing on the postman, I think the result for this API is actually matching the Flight Offer Search Model.
Besides, you might be confused why I add lots of new properties into this Model. The reason is that actually the model is supposed to contain these new properties cause the original model from [Flight Offer Search API ](https://developers.amadeus.com/self-service/category/air/api-doc/flight-offers-search/api-reference) also contains those properties.
![image](https://user-images.githubusercontent.com/74473717/159784453-d97fc240-aa5d-4729-bf53-08b2b39bfb41.png)

Reply:
And you can also find the data model is actually the Flight-Offer in the result of [Branded fares upsell API](https://developers.amadeus.com/self-service/category/air/api-doc/branded-fares-upsell/api-reference).
![image](https://user-images.githubusercontent.com/74473717/159785754-9ccf507e-2acb-44a0-993c-08ecbe261173.png)